src/ja/server/dispatcher/dispatcher.py

```diff
@@ -1,9 +1,14 @@
 from ja.common.job import JobStatus
 from ja.server.database.database import ServerDatabase
+from ja.server.database.types.job_entry import DatabaseJobEntry
+from ja.server.database.types.work_machine import WorkMachineState
 from ja.server.dispatcher.proxy_factory import WorkerProxyFactoryBase
 from typing import Dict
+from paramiko.ssh_exception import SSHException
```

Comment:
This import doesn't work for some reason

Reply:
I think you mean that mypy complains about the import.
You can fix this by adding `  #type: ignore` after the import.